mmv1/third_party/terraform/services/compute/resource_compute_router_test.go.tmpl

```diff
@@ -387,3 +426,39 @@ resource "google_compute_router" "foobar" {
 }
 `, routerName, routerName)
 }
+
+
+{{- if ne $.TargetVersionName "ga" }}
+func testAccComputeRouter_resourceManagerTags(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+		resource "google_compute_network" "foobar" {
+		provider				= google-beta
+		name                    = "%{network_name}"
+		auto_create_subnetworks = false
+		}
+
+		resource "google_compute_subnetwork" "foobar" {
+		provider                = google-beta
+		name          = "%{subnet_name}"
+		network       = google_compute_network.foobar.self_link
+		ip_cidr_range = "10.0.0.0/16"
+		region        = "%s"
```

Comment:
We need to fix the value, use `context`, or omit the field. I'd recommend omitting it, and using the test default.